SparkPost/2web2ui
# SA-2211 - (UI) Add missing integration tests for all filter and comparison types

[SA-2211](https://sparkpost.atlassian.net/browse/SA-2211)

### What Changed

- Adds a new Cypress-specific configuration array used to drive tests in both the compare by and filters form specs

### How To Test

- Verify tests adequately verify the I/O behavior for the filters form and the compare by form

### To Do
